tests/cpp/unit-tests/CMakeLists.txt

```diff
@@ -65,6 +65,21 @@ set(UNIT_TESTS
     test_redisserver.cpp
 )
 
+# Compare the number of files given above to the number of CPP files
+# in the directory. If the two numbers aren't equal, we're not testing
+# coverate completely. It is not recommended to use globbed file lists
+# in add_executable()
+list(LENGTH SOURCES source_count)
+
+file(GLOB cpp_files ../../../src/cpp/*.cpp)
+
+list(LENGTH cpp_files cpp_file_count)
+
+if(NOT (source_count EQUAL cpp_file_count))
+	message(FATAL_ERROR "CPP test coverage sources (${source_count}) does not match \
```

Comment:
Ignore this if you want, but mentioning that the first number for the CPP test coverage sources is the count might make the message more clear e.g. "CPP test coverage source count ${source_count}..." or "The count of CPP test coverage sources ${source_count}..."